### PR TITLE
Translate to BETA 100%

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-browser/issues/24471

PR to main: https://github.com/brave/brave-variations/pull/325

Main issue: https://github.com/brave/brave-browser/issues/23538

### CODE REVIEW

**Review by iefremov — APPROVED** [review]

**Review by linhkikuchi — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/309#issuecomment-1194562568.

**Comment by bsclifton** [comment]
Quick heads up - this accidentally set the wrong value. The PR to main (https://github.com/brave/brave-variations/pull/325) did set the correct value

We can keep this change though- as it closes out https://github.com/brave/brave-browser/issues/24546

**Pull request merged**
